src/encoded/static/components/forms/ExcelSubmissionView.js

```diff
@@ -765,10 +765,14 @@ function useInterval(callback, delay) {
     }, [delay]);
 }
 
+/**
+ * Note: by default Poller now uses datastore=database calls - this is to avoid bugs related to the late indexing of updated submissions
+ * and should not cause performance issues since there are limited items embedded onto ingestion submission items.
+ */
 function Poller(props){
     const { context = null, setStatusIdx, onLoadedIngestionSubmission, setIsSubmitting, pushNewAlert, clearAllAlerts } = props;
     const { uuid } = context || {};
-    const getURL = "/ingestion-submissions/" + uuid;
+    const getURL = "/ingestion-submissions/" + uuid + "?frame=object&datastore=database";
```

Comment:
I think there should be an extra "/" in here to prevent a redirect:
```suggestion
    const getURL = "/ingestion-submissions/" + uuid + "/?frame=object&datastore=database";
```